src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -51,11 +51,15 @@ public InputOutputColumnPair(string outputColumnName, string inputColumnName = n
     public static class TransformExtensionsCatalog
     {
         /// <summary>
-        /// Copies the input column to another column named as specified in <paramref name="outputColumnName"/>.
+        /// Creates a <see cref="ColumnCopyingEstimator"/> that copies the data from the column specified in <paramref name="inputColumnName"/>
```

Comment:
> that [](start = 58, length = 5)

'that' -> ', which' ?

Reply:
this that is valid, unless is a person. Then we need to use which? @natke?

---
In reply to: [275065191](https://github.com/dotnet/machinelearning/pull/3316#discussion_r275065191) [](ancestors = 275065191)

Reply:
This is one that always gets me. Strictly, "that" is used when you are defining the thing you are talking about, and "which" is used when you are giving extra information about the thing. In this case I would say we are in the second category because we know we are talking about a ColumnCopyEstimator. That is a very long-winded way of saying my vote goes with "Creates a <see cref="ColumnCopyingEstimator"/>### **, which** copies the data from the column specified in <paramref name="inputColumnName"/>"

Reply:
thanks for clearing it!

---
In reply to: [275080289](https://github.com/dotnet/machinelearning/pull/3316#discussion_r275080289) [](ancestors = 275080289)